msg-transport/src/quic/mod.rs

```diff
@@ -111,7 +111,8 @@ impl Transport<SocketAddr> for Quic {
         let endpoint = if let Some(endpoint) = self.endpoint.clone() {
             endpoint
         } else {
-            let Ok(endpoint) = self.new_endpoint(None, None) else {
+            let endpoint_addr = SocketAddr::unspecified_compatible_with(&addr);
```

Comment:
This would be replaced with 
```suggestion
            let endpoint_addr = &addr.as_unspecified();
```